### PR TITLE
Use getBytes(StandardCharsets.UTF_8) instead of getBytes("UTF-8")

### PR DESCRIPTION
**Pull request by wborn**
Follow up to #5083

### CODE REVIEW

File: addons/binding/org.openhab.binding.silvercrestwifisocket/src/main/java/org/openhab/binding/silvercrestwifisocket/internal/utils/WifiSocketPacketConverter.java [hunk 1]
```diff
@@ -268,7 +269,7 @@ private static String byteArrayToHexString(final byte[] array) {
     /**
      * Converts one partial byte array to one hexadecimal string.
      *
-     * @param array the byte array to convert.
+     * @param array  the byte array to convert.
```
Comment (cweitkamp) [inline addons/binding/org.openhab.binding.silvercrestwifisocket/src/main/java/org/openhab/binding/silvercrestwifisocket/internal/utils/WifiSocketPacketConverter.java]: Did you setup a new IDE? Looks like something had happened to your formatter. I am fine with ignoring it. Wdyt?

Comment (wborn) [reply addons/binding/org.openhab.binding.silvercrestwifisocket/src/main/java/org/openhab/binding/silvercrestwifisocket/internal/utils/WifiSocketPacketConverter.java]: It has been an issue for a while: https://github.com/eclipse/smarthome/issues/6424

Comment (wborn) [reply addons/binding/org.openhab.binding.silvercrestwifisocket/src/main/java/org/openhab/binding/silvercrestwifisocket/internal/utils/WifiSocketPacketConverter.java]: My auto formatter works again as it should, after applying the [easy fix]( https://github.com/eclipse/smarthome/issues/6424#issuecomment-471728524).

**Comment by cweitkamp** [comment]
Thanks.

**Pull request merged**